etl-jobs/ingestion-spec/vdn_content_index_batch.json

```diff
@@ -0,0 +1,816 @@
+{
+    "type":"index",
+    "spec":{
+      "dataSchema":{
+        "dataSource":"vdn-content-model-snapshot",
+        "parser":{
+          "type":"string",
+          "parseSpec":{
+            "format":"json",
+            "flattenSpec": {
+              "useFieldDiscovery": false,
+              "fields": [
+                {
+                  "type": "path",
+                  "name": "author",
+                  "expr": "$.data.author"
+                },
+                {
+                  "type": "path",
+                  "name": "board",
+                  "expr": "$.data.board"
+                },
+                {
+                  "type": "path",
+                  "name": "lastStatusChangedOn",
+                  "expr": "$.data.lastStatusChangedOn"
+                },
+                {
+                  "type": "path",
+                  "name": "collectionId",
+                  "expr": "$.data.collectionId"
+                },
+                {
+                  "type": "path",
+                  "name": "organisationId",
+                  "expr": "$.data.organisationId"
+                },
+                {
+                  "type": "path",
+                  "name": "acceptedContents",
+                  "expr": "$.data.acceptedContents"
+                },
+                {
+                  "type": "path",
+                  "name": "rejectedContents",
+                  "expr": "$.data.rejectedContents"
```

Comment:
Add a transform spec and convert the array to sizes - acceptedContentsSize and rejectedContentsSize